Add section on includeLevel to readme

martin3walker opened the pull request:
Include a section about how to return links more than one level deep. Useful because Contentful advocates for structuring content in a way the makes use of many references ([see article](https://www.contentful.com/r/knowledgebase/topics-and-assemblies/)). So, this seems like a problem many people could run into if they're reading Contentful's docs.

martin3walker committed:
Add section on includeLevel to readme

Include a section about how to return links more than one level deep. Useful because the content structure advocated for by Contentful makes heavy use of references. So, this seems like a problem many people could run into.

jescalan commented on README.md:
```diff
@@ -117,6 +117,31 @@ new Contentful({
+
+Contentful allows you to link content types with a field called references. This can create a chain of links. For example __Content Entry A__ could contain a reference to __Content Entry B__, and __Content Entry B__ could further reference __Content Entry C__. 
+
+Spike's default settings return only the first level of links, meaning that returning __Content Entry A__ from above will also retun __Content Entry B__, but not __Content Entry C__. There is an option called includeLevel that can be included in your Contentful object, which will allow you to return more levels of links than the default 1. This is extremely helpful if your content model relies he ...
```
So this is not specific to spike, it is actually contentful's default, which we do not change out of respect for the choices they made with their client. I think it might make it more accurate if it was changed to "Contentful's default settings..."

jescalan reviewed (changes requested):
This is fantastic, thank you for the contribution! Just one small adjustment I noticed and we can get this merged.

martin3walker committed:
Added desired changes to PR

Correctly described the default link resolution settings as Contentful's, rather than Spike's.

jescalan commented:
👌 looks great - thanks again for contributing!

martin3walker commented:
Thanks for a great static site generator!
